app/src/main/java/xyz/whereuat/whereuat/KeyLocationUtils.java

```diff
@@ -21,6 +66,36 @@ public static boolean locIsValid(Location loc) {
         return loc != null;
     }
 
+    public static KeyLocation findNearestLoc(Cursor c, Location origin) throws
```

Comment:
Can you add a comment here about `c` being a cursor to a query over key locations from the db?